lib/active_merchant/billing/gateways/global_collect.rb

```diff
@@ -2,13 +2,17 @@ module ActiveMerchant #:nodoc:
   module Billing #:nodoc:
     class GlobalCollectGateway < Gateway
       class_attribute :preproduction_url
+      class_attribute :ogone_direct_test
+      class_attribute :ogone_direct_live
```

Comment:
Why is this being added to the `global_collect` gateway if it's ogone?

Reply:
It is a little weird,  as far I know  `Ogone` and `GlobalCollect` has the same PSP, `Ogone` has two types of integration Classic/Direct that's why we are using GlobalCollect gateway to connect the Direct API for `Ogone`, because the `GlobalCollect` API seems to have a similar integration for the Direct API